pkgs/development/interpreters/ruby/gemconfig/default.nix

```diff
@@ -42,10 +42,16 @@ in
   };
 
   libv8 = attrs: {
-    buildInputs = [ which v8 python ];
+    dontBuild = false;
+    buildInputs = [ which v8 python git ];
     buildFlags = [
       "--with-system-v8=true"
     ];
+    # due to `git ls-files` ...
```

Comment:
I believe you're correct. I'll make that the default, and also fix up the other things you've pointed out. Thanks for reviewing this :).